### ISSUE TITLE
Update cleo version

### ISSUE THREAD

**Issue by NicolaSmaniotto** [iamkroot/trakt-scrobbler#240]
Is it possible to depend on a more recent release of cleo? Gentoo recently removed 0.8.1 from the main repository, so this program can no longer be used.

**Comment by iamkroot** [comment]
This one is going to be tricky. They have made a _lot_ of breaking changes in v1, many of which will require significant work to upgrade.

At this point, I'm wary of investing further in `cleo` because of how poetry-specific it has become. Will try to see if there are any alternatives.

> this program can no longer be used.

You could always go with the recommended installation method of `pipx` and let venvs work their magic :)

**Comment by mjsir911** [comment]
Yeah running into this as well since my preference is distro-wide packages
